src/model/encoding/convertFromHTMLToContentBlocks.js

```diff
@@ -652,25 +688,16 @@ const convertChunkToContentBlocks = (chunk: Chunk): ?Array<BlockNodeRecord> => {
       text: textBlock,
       characterList,
       prevSibling:
-        index === 0 || rawBlocks[index - 1].parent !== parent
+        parentTextNodeKey ||
```

Comment:
I could use a bit more explanation for why we put the `parentTextNodeKey` as the prevSibling. It makes sense that it's what we want - like in the case of `<blockquote>hi <h1>world</h1></blockquote>`. I'll re-read this again and get more clarity.